src/mscorlib/src/System/String.cs

```diff
@@ -666,6 +666,37 @@ private unsafe String CtorCharPtrStartLength(char* ptr, int startIndex, int leng
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         public extern String(char c, int count);
 
+        private unsafe string CtorCharRefLength(ref char value, int length)
+        {
+            Debug.Assert(length >= 0);
+
+            if (length == 0)
+            {
+                return Empty;
+            }
+
+            try
+            {
+                string result = FastAllocateString(length);
+                fixed (char* dest = &result._firstChar, src = &value)
+                {
+                    wstrcpy(dest, src, length);
+                }
+                return result;
+            }
+            catch (NullReferenceException)
```

Comment:
I'm fine with that; I was just following suit from the other ctors.  I'll remove it.